scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
+// Meter
+// ==========================================================================
+meter {
+	-webkit-appearance: none;
+	width: 550px;
+	height: 25px;
+}
+
+.meter::-webkit-meter-bar, .multiColor::-webkit-meter-bar, .stripMeter::-webkit-meter-bar {
+	box-shadow: inset 0 0 7px -2px rgba(0,0,0,0.35);
+	background: #f1f1f1;
```

Comment:
It would be nice to have colors be tied into the a color variables instead of defined here, to allow for easy customization.